ic-utils/src/interfaces/http_request.rs

```diff
@@ -95,6 +96,23 @@ impl<'agent> Canister<'agent, HttpRequestCanister> {
             .build()
     }
 
+    pub fn http_request_update<'canister: 'agent, M: Into<String>, U: Into<String>, B: AsRef<[u8]>>(
```

Comment:
I think it would be worth introducing `http_request_query` as well for consistency. We could keep `http_request` as a wrapper around that for backwards compatibility for now.

Reply:
> I think it would be worth introducing `http_request_query` as well for consistency. We could keep `http_request` as a wrapper around that for backwards compatibility for now.

Should we do this?